src/nanosvg.h

```diff
@@ -939,7 +938,7 @@ static void nsvg__addShape(NSVGparser* p)
 		return;
 
 	shape = (NSVGshape*)malloc(sizeof(NSVGshape));
-	if (shape == NULL) goto error;
+	if (shape == NULL) return;
```

Comment:
how is this new version better? it ignores the idiomatic style of the rest of the file.  as an external observer, i'd guess this makes the proposed patch less appetizing.

Reply:
It does not "ignore the idiomatic style of the rest of the file". There are other instances where return is done early. See for example `nsvgDelete`. 

`error` label was used only once here. And all it done was check if shape is not null (and free it), which is always false. It makes function more readable, without useless control flow.